packages/kbn-logging/src/ecs/elf.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * https://www.elastic.co/guide/en/ecs/1.12/ecs-elf.html
+ *
+ * @internal
+ */
+export interface EcsElf {
```

Comment:
<img src="https://user-images.githubusercontent.com/3198181/135587378-13ff4b9a-bdef-4008-8a0d-c2d7d1b7a4f9.jpg" width="350" height="350"/>

Reply:
More like

![download](https://user-images.githubusercontent.com/1532934/135601513-a9ccd3bd-8ed3-4c77-932b-12f7649c1ab8.png)

Reply:
should've included some Legolas asciiart in the code comments...